src/V3LinkInc.cpp

```diff
@@ -0,0 +1,239 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Replace increments/decrements with new variables
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2020 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+// V3LinkInc's Transformations:
+//
+// Expand each pre and post incrementation/decrementation into separate
+// statements.
```

Comment:
Please add a bit more information on what the traversal is searching for (V3Slice has good example) and what is created, thanks.